.idea/ivy.iml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="PYTHON_MODULE" version="4">
   <component name="NewModuleRootManager">
-    <content url="file://$MODULE_DIR$" />
+    <content url="file://$MODULE_DIR$">
```

Comment:
Please don't update files in the .idea folder. Can you remove those update from the PR?

Reply:
I didn't update them manually. i remember running  ./merge_with_upstream.sh master. in my git bash to update the master branch. when created a commit, I saw 5 files changed. 
thanks for the detailed correction. I will make the changes now